frontend/tests/features/collection/revision.test.ts

```diff
@@ -2,17 +2,24 @@ import { ElementHandle, expect, Page, test } from "@playwright/test";
 import { ROUTES } from "src/common/constants/routes";
 import { TEST_URL } from "tests/common/constants";
 import {
-  describeIfDeployed,
   getInnerText,
   goToPage,
+  isDevStaging,
   login,
   tryUntil,
 } from "tests/utils/helpers";
 import { getTestID, getText } from "tests/utils/selectors";
 
+const { describe, skip } = test;
+
 const COLLECTION_ROW_ID = "collection-row";
 
-describeIfDeployed("Collection Revision", () => {
+describe("Collection Revision", () => {
+  skip(
+    !isDevStaging,
+    "We only seed published collections for revision test in dev and staging"
+  );
```

Comment:
There is a test coverage diff introduced here, these tests will no longer run on rdev and happy

Reply:
(this actually goes for all tests where we changed `describeIfDeployed` to `skip(!isDevStaging,...)`

Reply:
Yeah the change is intentional because revision depends on existing published collections. which I think BE only seeds published collections in dev and staging, thus not available in happy local container or rdev!